docs/design/schedule_algorithm.md

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Sorry, never mind, just read your second sentence. :)

But note that the location has moved to docs/devel/scheduler.md